odev/common/databases/local.py

```diff
@@ -449,6 +449,21 @@ def neutralize(self):
         """Neutralize the database."""
         assert self.process is not None, "Database process is not set"
         assert self.version is not None, "Database version is not set"
+        admin = self._db_user_admin()
+
+        if not admin["active"]:
+            logger.warning(
+                "The administrator account is archived in this database, this will cause issues during neutralization\n"
+                "Reactivating the administrator account now, please check access rights and other features "
+                "after neutralization"
```

Comment:
I would not talk about checking the right but more about checking the project on {p|s}aas cause it shouldn't be the case and can lead to some issue in the production database , no ?